dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBContextInitializer.java

```diff
@@ -70,6 +70,7 @@ class JAXBContextInitializer extends ServiceModelVisitor {
     private Set<Class<?>> globalAdapters = new HashSet<>();
     private Map<String, Object> unmarshallerProperties;
     private Bus bus;
+    private boolean isLoggableFinest = LOG.isLoggable(Level.FINEST);  // Liberty change
```

Comment:
Should this be `final`? `static`?

Reply:
On further thought, I don't think we should cache this check at the class level. If we do, we can't dynamically update the trace without restarting the server/application.

You could cache it at the start of a trace heavy method if performance is concerned.

Reply:
Good point, we need to come up with a balanced approach that would not cause performance problems and at the same time support dynamic tracing. I will do some testing with dynamic tracing to see what is the best approach here